db/schema.rb

```diff
@@ -162,41 +162,41 @@
     t.datetime "updated_at"
   end
 
-  create_table "stages", force: true do |t|
-    t.string   "name",                                        null: false
-    t.integer  "project_id",                                  null: false
+  create_table "stages", force: :cascade do |t|
+    t.string   "name",                                                                       null: false
+    t.integer  "project_id",                                                                 null: false
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "notify_email_address"
     t.integer  "order"
     t.datetime "deleted_at"
-    t.boolean  "confirm",                     default: true
+    t.boolean  "confirm",                                                    default: true
     t.string   "datadog_tags"
     t.boolean  "update_github_pull_requests"
-    t.boolean  "deploy_on_release",           default: false
+    t.boolean  "deploy_on_release",                                          default: false
     t.boolean  "comment_on_zendesk_tickets"
-    t.boolean  "production",                  default: false
+    t.boolean  "production",                                                 default: false
     t.boolean  "use_github_deployment_api"
-    t.string   "permalink",                                   null: false
-    t.text     "dashboard",                   limit: 65535
-    t.boolean  "email_committers_on_automated_deploy_failure",         default: false, null: false
-    t.string   "static_emails_on_automated_deploy_failure", limit: 255
+    t.string   "permalink",                                                                  null: false
+    t.text     "dashboard",                                    limit: 65535
+    t.boolean  "email_committers_on_automated_deploy_failure",               default: false, null: false
+    t.string   "static_emails_on_automated_deploy_failure",    limit: 255
     t.string   "datadog_monitor_ids",                          limit: 255
   end
 
-  add_index "stages", ["project_id", "permalink", "deleted_at"], name: "index_stages_on_project_id_and_permalink_and_deleted_at", using: :btree
+  add_index "stages", ["project_id", "permalink", "deleted_at"], name: "index_stages_on_project_id_and_permalink_and_deleted_at"
 
-  create_table "stars", force: true do |t|
+  create_table "stars", force: :cascade do |t|
     t.integer  "user_id",    null: false
     t.integer  "project_id", null: false
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
-  add_index "stars", ["user_id", "project_id"], name: "index_stars_on_user_id_and_project_id", unique: true, using: :btree
+  add_index "stars", ["user_id", "project_id"], name: "index_stars_on_user_id_and_project_id", unique: true
 
-  create_table "users", force: true do |t|
-    t.string   "name"
+  create_table "users", force: :cascade do |t|
+    t.string   "name",                           null: false
```

Comment:
Did you mean to add a non-null constraint to this column? The Travis tests are failing as a result of this.